### PR TITLE
open-policy-agent: init at 0.15.0

### PR DESCRIPTION
**Pull request by nlewo**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Review by tomberek — APPROVED** [review]
Builds and runs on NixOS. Evaluation looks good.

File: pkgs/development/tools/open-policy-agent/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "open-policy-agent";
+  version = "v0.9.2";
```
Comment (c0bw3b) [inline pkgs/development/tools/open-policy-agent/default.nix]:
```suggestion
  version = "0.9.2";
```
Version [must start with a digit](https://nixos.org/nixpkgs/manual/#sec-package-naming).

File: pkgs/development/tools/open-policy-agent/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  goPackagePath = "github.com/open-policy-agent/opa";
+  src = fetchFromGitHub {
+    owner = "open-policy-agent";
+    repo = "opa";
+    rev = version;
```
Comment (c0bw3b) [inline pkgs/development/tools/open-policy-agent/default.nix]:
```suggestion
    rev = "v${version}";
```

**Review by c0bw3b — CHANGES_REQUESTED** [review]
Upstream seems to be at v0.15.0 now.

File: pkgs/development/tools/open-policy-agent/default.nix (hunk 1 above)
Comment (nlewo) [reply pkgs/development/tools/open-policy-agent/default.nix]: Oops, I know it moreover :/

**Comment by nlewo** [comment]
@c0bw3b Thx for your review. I applied all your suggestions.

**Review by c0bw3b — APPROVED** [review]
Builds and runs

**Pull request merged**